### DISCUSSION TITLE
Include all submissions when exporting

### DISCUSSION THREAD

**Discussion by racuna1** [gradescope/autograder_samples#91] [Ideas]
It would be nice if there was an option to download all previous code submissions. (The current behavior of export is to provide only the most recent submission.) This is useful for analyzing the trace of actions that a student used to complete an assignment. Although the majority of the trace is provided through the test cases in the submissions' metadata, having access to the raw files allows for additional static analysis.

**Comment by ibrahima** [comment]
Thanks for the suggestion @racuna1! This is definitely a request that we've gotten before too. We have recently revamped our public roadmap where users can submit ideas. Please take a look at https://www.gradescope.com/roadmap to see if we already have what you're looking for, and if not, go ahead and submit the idea. That way our product team can catalog and prioritize feature requests accordingly. Thank you!

**Reply by racuna1** [reply]
Will do, thanks!
